src/utils/validation.ts

```diff
@@ -31,6 +31,10 @@ export const validation = {
     return guidRegEx.test(guid);
   },
 
+  isValidUserPrincipalNameArray(ups: string[]): boolean {
```

Comment:
Since we're passing here multiple values, it would be more user-friendly, if we said which one is invalid. Otherwise the user needs to guess through all of them and guess which one we consider invalid.

Reply:
Maybe we should do this for `isValidGuidArray` as well then.

Reply:
Good idea. Let's take it separately